crates/cli/src/subcommands/describe.rs

```diff
@@ -30,12 +30,7 @@ pub fn cli() -> clap::Command {
                 .long_help("The identity to use to describe the entity. If no identity is provided, the default one will be used."),
         )
         .arg(
-            Arg::new("anon_identity")
-                .long("anon-identity")
-                .short('a')
-                .conflicts_with("identity")
-                .action(SetTrue)
-                .help("If this flag is present, no identity will be provided when describing the database"),
+            common_args::anonymous()
```

Comment:
removed the `.conflicts_with("identity")` because the identity param is already marked as conflicting with this one